src/test/scala/VerifierSpec.scala

```diff
@@ -212,6 +212,9 @@ class BasicVerifierSpec extends AnyFlatSpec {
   "test-multiply-divide.ucl" should "verify all but one assertions." in {
     VerifierSpec.expectedFails("./test/test-multiply-divide.ucl", 1)
   }
+  "test-multiply-divide-subtract-chaining.ucl" should "verify all assertions." in {
+    VerifierSpec.expectedFails("./test/test-multiply-divide-subtract-chaining.ucl", 0)
```

Comment:
```suggestion
    VerifierSpec.expectedFails("./test/test-multiply-divide-subtract-chaining.ucl", 1)
```
(see suggested addition to test)